tests/e2e/idps_test.go

```diff
@@ -0,0 +1,88 @@
+package e2e
+
+import (
+	"net/http"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	conn "github.com/terraform-redhat/terraform-provider-rhcs/tests/ci"
+	"github.com/terraform-redhat/terraform-provider-rhcs/tests/utils/cms"
+	con "github.com/terraform-redhat/terraform-provider-rhcs/tests/utils/constants"
+	exe "github.com/terraform-redhat/terraform-provider-rhcs/tests/utils/exec"
+)
+
+var _ = Describe("TF Test", func() {
+	Describe("Identity Providers test cases", func() {
+
+		// all identity providers - declared for future cases
+		type IDPServices struct {
```

Comment:
nit. Why do we need this? if we have a closure for each IDP type, we can have a variable in each one for the IDP under testing. But again, not a dealbreaker.

Reply:
tnx! will be handled in the next PR for idps